<launch>
<node name="left_robotiq_gripper_3_finger_adaptive_state" pkg="robotiq_gripper_3_finger_adaptive_control" type="left_robotiq_gripper_3_finger_adaptive_state" />

<node name="right_robotiq_gripper_3_finger_adaptive_state" pkg="robotiq_gripper_3_finger_adaptive_control" type="right_robotiq_gripper_3_finger_adaptive_state" />
</launch>
